ray-operator/config/samples/ray-service.autoscaler.yaml

```diff
@@ -9,72 +9,56 @@ metadata:
 spec:
   serviceUnhealthySecondThreshold: 300 # Config for the health check threshold for service. Default value is 60.
   deploymentUnhealthySecondThreshold: 300 # Config for the health check threshold for deployments. Default value is 60.
-  serveConfig:
-    importPath: fruit.deployment_graph
-    runtimeEnv: |
-      working_dir: "https://github.com/ray-project/test_dag/archive/41d09119cbdf8450599f993f51318e9e27c59098.zip"
-    deployments:
-      - name: MangoStand
-        numReplicas: 1
-        userConfig: |
-          price: 3
-        rayActorOptions:
-          numCpus: 1
-      - name: OrangeStand
-        numReplicas: 1
-        userConfig: |
-          price: 2
-        rayActorOptions:
-          numCpus: 0.1
-      - name: PearStand
-        numReplicas: 1
-        userConfig: |
-          price: 1
-        rayActorOptions:
-          numCpus: 0.1
-      - name: FruitMarket
-        numReplicas: 1
-        rayActorOptions:
-          numCpus: 0.1
-      - name: DAGDriver
-        numReplicas: 1
-        routePrefix: "/"
-        rayActorOptions:
-          numCpus: 0.1
+  serveConfigV2: |
+    applications:
+      - name: app1
+        import_path: autoscaling_test.blocked.app
+        route_prefix: /
+        runtime_env:
+          working_dir: "https://github.com/ray-project/serve_workloads/archive/7a0daafdb86b55118534b04d122b1b79c9ae99f5.zip"
+        deployments:
+          - name: Blocked
+            autoscaling_config:
+              metrics_interval_s: 0.1
+              min_replicas: 1
+              max_replicas: 10
+              look_back_period_s: 0.2
+              downscale_delay_s: 0
+              upscale_delay_s: 0
+            graceful_shutdown_timeout_s: 1
+            max_concurrent_queries: 1000
+            ray_actor_options:
+              num_cpus: 0.5
+      - name: signal
+        import_path: autoscaling_test.signaling.app
+        route_prefix: /signal
+        runtime_env:
+          working_dir: "https://github.com/ray-project/serve_workloads/archive/7a0daafdb86b55118534b04d122b1b79c9ae99f5.zip"
   rayClusterConfig:
     rayVersion: '2.5.0' # should match the Ray version in the image of the containers
     ## raycluster autoscaling config
     enableInTreeAutoscaling: true
     autoscalerOptions:
       upscalingMode: Default
-      resources:
-        limits:
-          cpu: "1"
-          memory: "1000Mi"
-        requests:
-          cpu: "1"
-          memory: "1000Mi"
     ######################headGroupSpecs#################################
     headGroupSpec:
       # The `rayStartParams` are used to configure the `ray start` command.
       # See https://github.com/ray-project/kuberay/blob/master/docs/guidance/rayStartParams.md for the default settings of `rayStartParams` in KubeRay.
       # See https://docs.ray.io/en/latest/cluster/cli.html#ray-start for all available options in `rayStartParams`.
-      rayStartParams: {"num-cpus": "0"}
+      rayStartParams:
```

Comment:
What is the reason for this change?

Reply:
I didn't make the changes from the original autoscaler file, instead I added autoscaling configurations to the original `config/samples/ray_v1alpha1_rayservice.yaml` file.